csro/oiwfs-adc-assembly/publish-model.conf

```diff
@@ -237,7 +233,6 @@ publish {
 
       The ADC will shift an image position in a manner that depends upon the current state of the ADC. The estimated image shift is published by IRIS for the TCS so that the TCS can adjust the IRIS OIWFS position demands accordingly. The reference wavelength used to calculate the shift for each of the WFS is tcs.cm.iris.oiwfsXAtmDispersion.referenceWavelength.
       """
-      minRate = 1
```

Comment:
why are the `minRate` values being removed? is it worth writing comments in the files to explain changes?

Reply:
Good question. I went and looked... minRate was explicitly removed from the schema last December. I guess it wasn't useful to them: https://github.com/tmtsoftware/icd/commit/b17df023cd9cd7c37ccc685266ae114076cd5a08

Reply:
ok, it would be useful to have this information included in the commit message about this change so that it becomes traceable.

Reply:
I guess I could add it as a general comment (along with the other changes described in this PR) in the final merge commit of this branch?